examples/generic-st77xx-with-xpt2046.py

```diff
@@ -0,0 +1,50 @@
+import sys
+sys.path.append('.')
+from st77xx import *
+from xpt2046 import *
+
+
+spi=machine.SPI(
+    1,
+    baudrate=24_000_000,
+    polarity=0,
+    phase=0,
+    sck=machine.Pin(10,machine.Pin.OUT),
+    mosi=machine.Pin(11,machine.Pin.OUT),
+    miso=machine.Pin(12,machine.Pin.IN)
+)
+
+## TODO: measure fps gain w/ DMA
+if 0:
+    # with DMA, the repaints seem to be too slow? To be investigated
+    # but we seem to be fine performance-wise without DMA anyway
+    import rp2_dma
+    rp2_dma=rp2_dma.DMA(0)
+else: rp2_dma=None
+
+import lvgl as lv
+lv.init()
+
+# create async event loop; this example freezes with the default sync (timer-based) event loop created in the driver (why?)
+import lv_utils
+import uasyncio
+lv_utils.event_loop(refresh_cb=lv.task_handler,asynchronous=True)
```

Comment:
You shouldn't set `refresh_cb=lv.task_handler` here.  
This is [done internally](https://github.com/lvgl/lv_binding_micropython/blob/28109f072a05c35de67b21edf20c7300a7ee9ceb/lib/lv_utils.py#L119) in event_loop.   
`refresh_cb` is usually not needed, except from cases where you need some hook to refresh the screen [for example in SDL](https://github.com/lvgl/lv_binding_micropython/blob/28109f072a05c35de67b21edf20c7300a7ee9ceb/lib/lv_utils.py#L12) but this is not needed for most display drivers.

Reply:
I removed all that setup from the `generic-st77xx-with-xpt2046.py` and it works better now.

Reply:
Done as you suggested. It works better now :)